publish.js

```diff
@@ -10,20 +10,24 @@ You may obtain a copy of the License at
 https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
 */
 
+/* eslint-env node */
+
 "use strict";
 
 var publish = {};
 var path = require("path");
 var extend = require("extend");
 
+// TODO: The supported version of node.js does not yet support ES6 template strings
+// When version node.js 4.x.x is supported this can be replaced by native support.
+var es6Template = require("es6-template-strings");
```

Comment:
Leaving in es6-template-strings based on our conversation in the channel.
https://botbot.me/freenode/fluid-work/2016-08-30/?msg=72191253&page=1